components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthComponentServiceHolder.java

```diff
@@ -47,6 +48,28 @@
     private List<TokenBindingMetaDataDTO> tokenBindingMetaDataDTOs = new ArrayList<>();
     private OAuthAdminServiceImpl oAuthAdminService;
 
+    public List<ScopeValidator> getScopeValidators() {
+
+        return scopeValidators;
+    }
+
+    public void addScopeValidator(ScopeValidator scopeValidator) {
+
+        scopeValidators.add(scopeValidator);
+    }
+
+    public void removeScopeValidator(ScopeValidator scopeValidator) {
+
+        scopeValidators.remove(scopeValidator);
+    }
+
+    public void setScopeValidators(List<ScopeValidator> scopeValidators) {
+
+        this.scopeValidators = scopeValidators;
+    }
+
+    private List<ScopeValidator> scopeValidators = new ArrayList<>();
```

Comment:
Lets move the variable to the top. Would be cleaner.

Reply:
moved in 79d44c4e3bce8a5f1c57e5dce757c1bb19337031